src/ragas/llms/prompt.py

```diff
@@ -136,7 +147,7 @@ def get_example_str(self, example_no: int) -> str:
                 else value
             )
             example_str += f"\n{key}: {value}"
-        return example_str
+        return "```" + example_str + "```"
```

Comment:
why do we use the triple backticks ?
other models seem to mess this up by adding extra tokens like ```bash etc

Reply:
if I am not mistaken, the json/pydantic response parser in langchain search for the triple backsticks in the response to identify the json structure in the sometimes more verbose response of the model. Some models love to add some free text preamble before the json block, or conclusion/explanation of their work done after the json.